ci_environment/mysql/templates/default/my.cnf.erb

```diff
@@ -51,14 +51,17 @@ basedir         = /usr
 datadir         = <%= node['mysql']['data_dir'] %>
 tmpdir          = /tmp
 skip-external-locking
+
 #
 # Instead of skip-networking the default is now to listen only on
 # localhost which is more compatible and is not less secure.
 bind-address            = <%= node['mysql']['bind_address'] %>
 
+innodb_file_per_table  = true
+innodb_file_format     = Barracuda
```

Comment:
Ah, I see. `innodb_file_per_table` is necessary because [Barracuda](http://dev.mysql.com/doc/innodb/1.1/en/glossary.html#glos_barracuda) requires it.

So, the question becomes "What is the advantage of the new format?"